lib/iron_worker_ng/feature/python/merge_requirements.rb

```diff
@@ -0,0 +1,35 @@
+module IronWorkerNG
+  module Feature
+    module Python
+      module MergeRequirements
+        module InstanceMethods
+
+          def merge_requirements(path)
+            feature = IronWorkerNG::Feature::Base.new(self)
+            IronWorkerNG::Fetcher.fetch_to_file(feature.rebase(path)) do |requirements|
```

Comment:
First wanted to pass requirements directly without parsing  `pip install -r -requirements.txt` but with current version of code it would be hard to change something